<launch>
  <node name="under_carriage_4wheel" pkg="harurobo2022" type="under_carriage_4wheel" output="screen" />
  <node name="can_subscriber_node" pkg="harurobo2022" type="can_subscriber_node" output="screen" />
  <node name="emergency_controller" pkg="harurobo2022" type="emergency_controller" output="screen" />
  <node name="manual_commander" pkg="harurobo2022" type="manual_commander" output="screen" />
  <node name="state_manager" pkg="harurobo2022" type="state_manager" output="screen" />
</launch>